src/sass/app_2/modules/trading.scss

```diff
@@ -324,6 +324,89 @@
                 right: 39%;
             }
         }
+        .has-currency-options, .currency-options {
```

Comment:
this file is becoming too long, can we put styles for expandable input into its own file rather than under the trading class?